src/main/java/com/example/parser/utils/Parser.java

```diff
@@ -14,13 +16,13 @@
 
 public class Parser {
 
-    // private static final String filePath = "C:\\Users\\Любомир\\Downloads\\dozvoli-na-vikidi-v-atmosferu-onovlenii.json";
     private PermitForEmissionsOfPollutants permit;
     private final PermitRepository permitRepository;
     private float time;
     private Date date;
     private int File_counter;
     private int DB_counter;
+    private static Logger log = LoggerFactory.getLogger(Parser.class);
```

Comment:
Make it `private static final` as constant

Reply:
done